### PR TITLE
new PPSh and mosin sprite, Paladin-12 widening because too small. 

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
gives PPSh and Mosin a new sprite, paladin-12 gets their sprite a bit bigger due to them being too small compared to all other guns.

also there's a small change to typepath, for organizing things, makes irremoveable a type of stock, instead of being a subtype of rifle stock.

image

## Why It's Good For The Game
good sprite

## Changelog
:cl:
imageadd: added new mosin, ppsh sprite and widened Paladin-12 sprite
/:cl:

### CODE REVIEW

**Comment by Smudged** [comment]
When I was talking over on the discord, I just realised that the mosin you sprited looks alooot more like a M1 Garand

From my perspective it's because of the two black bands which makes it more like a garand than mosin, and the wood coloration too

**Review by TheBimmer231 — CHANGES_REQUESTED** [review]
shotgun needs to have more contrast, especially on the dark side, as for the PPSH, talk to me about the barrel.

**Comment by TheBimmer231** [comment]
Just realized the mosin is outrageously long too ⚡

**Comment by SplinterGP** [comment]
> Just realized the mosin is outrageously long too ⚡

the mosin without being long would look quite odd, since it was also well, long before, and as it is a mosin, but i'll look.

**Comment by SplinterGP** [comment]
image
image

File: code/modules/projectiles/guns/smgs.dm [hunk 1]
```diff
@@ -108,6 +108,9 @@
 		/obj/item/attachable/heavy_barrel,
 		/obj/item/attachable/scope/mini,
 		/obj/item/attachable/magnetic_harness,
+		/obj/item/attachable/attached_gun/shotgun,
+		/obj/item/attachable/attached_gun/grenade,
+		/obj/item/attachable/attached_gun/flamer,
```
Comment (Hughgent) [inline code/modules/projectiles/guns/smgs.dm:113]: undocumented balance change. add to CL

**Review by Hughgent — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/smgs.dm (hunk 1 above)
Comment (SplinterGP) [reply code/modules/projectiles/guns/smgs.dm:113]: oh sorry

Comment (SplinterGP) [reply code/modules/projectiles/guns/smgs.dm:113]: i forgot that in

**Comment by spookydonut** [comment]
Remove the balance changes and this can be merged

**Review by TheBimmer231 — CHANGES_REQUESTED** [review]
If you can't make the mosin shorter then remove it, otherwise I'm not approving this PR.

**Comment by SplinterGP** [comment]
Alright, will shorten mosin stock.

**Comment by SplinterGP** [comment]
> Remove the balance changes and this can be merged

Its done, the "balance" changes on mosin is just removing barrel and putting the stats from it on gun, since it used barrel for sprite

**Comment by Pariah919** [comment]
@SplinterGP remove the mosin barrel

**Review by TiviPlus — APPROVED** [review]

**Comment by SplinterGP** [comment]
closing until i get able to deal with it.

**Pull request closed without merging**